packages/openid4vc-client/tests/fixtures.ts

```diff
@@ -15,63 +22,50 @@ export const getMetadataResponse = {
   response_types_supported: ['code id_token', 'code', 'id_token', 'none'],
   scopes_supported: ['OpenBadgeCredential', 'AcademicAward', 'LearnerProfile', 'PermanentResidentCard'],
   token_endpoint_auth_signing_alg_values_supported: ['HS256', 'RS256', 'PS256', 'ES256', 'EdDSA'],
-  credential_endpoint: 'https://launchpad.vii.electron.mattrlabs.io/oidc/v1/auth/credential',
-  credentials_supported: {
-    OpenBadgeCredential: {
-      formats: {
-        ldp_vc: {
-          name: 'JFF x vc-edu PlugFest 2',
-          description: "MATTR's submission for JFF Plugfest 2",
-          types: ['OpenBadgeCredential'],
-          binding_methods_supported: ['did'],
-          cryptographic_suites_supported: ['Ed25519Signature2018'],
-        },
-      },
+  credentials_supported: [
```

Comment:
Restructured based on the specification. Would be nice if we can get some more official fixtures from other implementations using draft 11.